userspace/libpman/src/stats.c

```diff
@@ -140,10 +140,10 @@ int pman_get_scap_stats(struct scap_stats *stats)
 	return errno;
 }
 
-static void set_u64_monotonic_kernel_counter(uint32_t pos, uint64_t val)
+static void set_u64_monotonic_kernel_counter(uint32_t pos, uint64_t val, uint32_t metric_type)
```

Comment:
```suggestion
static void set_u64_monotonic_kernel_counter(uint32_t pos, uint64_t val, uint32_t metric_flag)
```
Since it clashes with the definition of 

```
g_state.stats[pos].metric_type = METRIC_VALUE_METRIC_TYPE_MONOTONIC;
```

Reply:
done, thank you!